azure-pipelines.yml

```diff
@@ -16,17 +16,17 @@ steps:
   displayName: 'Install Node.js'
 
 - script: |
-    npm install
-    npm run tslint
+    yarn install
+    yarn run tslint
     if [ $? -ne 0 ]; then
         exit 1
     fi
-    npm run compile 
+    yarn run compile
```

Comment:
yarn compile

Reply:
sure let me check, I never use yarn before

Reply:
you're right - will fix it in a second (working both ways though)